assets/styles/app.css

```diff
@@ -969,4 +969,66 @@ initiative-title-container {
 .card-faq {
     height: 11.25rem;
     width: 28.641rem;
-}
\ No newline at end of file
+}
+
+.body-not-found {
+    background-color: var(--bg-color-f);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 90%;
+    padding: 1.4rem 0 1rem;
+}
+
+.error-container {
+    width: 75%;
+    margin-bottom: 30px;
+    padding: 1.8rem;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3);
+    border-radius: 8px;
+    display: flex;
+    gap: 1.5rem;
+    align-items: center;
+}
+
+.error-text {
+    width: 50%;
+    font-family: var(--font-open-sans);
+    color: var(--main-font-color);
+    margin-left: 18rem;
+}
+
+.error-text h1 {
+    font-size: 45px;
+    font-weight: bold;
+}
+
+.error-text h2 {
+    font-size: 73px;
+    font-weight: bold;
+}
+
+.error-text p {
+    font-size: 17px;
+    margin-top: 1.7rem;
+}
+
+.btn-primary-error {
+    font-weight: bold;
+    background-color: var(--btn-publish-custom-color);
+    color: var(--bg-color-f);
+    margin-top: 1.8rem;
+    padding: 0.5rem 1rem;
+    font-size: 16px;
+    text-decoration: none;
+    display: inline-flex;
+    text-align: center;
+    border-radius: 5px;
+    margin-left: 0.2rem;
```

Comment:
não entendi o motivo desse button novo, se for aquele verde, apenas `btn btn-primary` já é pra funcionar